challenges/classes.js

```diff
@@ -4,4 +4,64 @@
 // console.log(cuboid.volume()); // 100
 // console.log(cuboid.surfaceArea()); // 130
 
-// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
\ No newline at end of file
+// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
+
+
+
+/* ===== Prototype Practice ===== */
+
+// Task: You are to build a cuboid maker that can return values for a cuboid's volume or surface area. Cuboids are similar to cubes but do not have even sides. Follow the steps in order to accomplish this challenge.
+
+/* == Step 1: Base Constructor ==
+  Create a constructor function named CuboidMaker that accepts properties for length, width, and height
+*/
+
+class CuboidMaker{
+    constructor(attr){
+        this.length = attr.length;
+        this.height = attr.height;
+        this.width = attr.width;
+    }
+    volume(){
+        const volume = this.length * this.width * this.height
+        return volume;
```

Comment:
Could do 

`return this.length * this.width * this.height`

 directly for less code.